src/main/java/uk/gov/digital/ho/hocs/queue/migration/MigrationService.java

```diff
@@ -1,21 +1,30 @@
 package uk.gov.digital.ho.hocs.queue.migration;
 
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import uk.gov.digital.ho.hocs.application.ClientContext;
 import uk.gov.digital.ho.hocs.client.casework.CaseworkClient;
+import uk.gov.digital.ho.hocs.client.casework.dto.CreateCaseworkCaseResponse;
+import uk.gov.digital.ho.hocs.client.casework.dto.CreateMigrationCaseRequest;
 import uk.gov.digital.ho.hocs.client.document.DocumentS3Client;
 import uk.gov.digital.ho.hocs.client.workflow.WorkflowClient;
+import uk.gov.digital.ho.hocs.client.workflow.dto.DocumentSummary;
+
+import java.util.List;
+import java.util.Map;
 
 @Slf4j
 @Service
 public class MigrationService {
 
+    public static final String CHANNEL_LABEL = "Channel";
+
      private final WorkflowClient workflowClient;
     private final CaseworkClient caseworkClient;
     private final ClientContext clientContext;
     private final DocumentS3Client documentS3Client;
-
+    @Autowired
```

Comment:
you shouldn't need Autowired with the Service annotation

Reply:
Removed